hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/upgrade/OneToZeroDowngrader.java

```diff
@@ -35,13 +35,13 @@
 /**
  * Downgrade handle to assist in downgrading hoodie table from version 1 to 0.
  */
-public class OneToZeroDowngradeHandler implements DowngradeHandler {
+public class OneToZeroDowngrader implements Downgrader {
```

Comment:
I’m not in favor of `<verb>-er` naming pattern.  We can discuss how such classes can be renamed.

Reply:
reverted